Genapi add bolt diy quickstart (#4787)
scaleway/docs-content: pages/generative-apis/reference-content/integrating-generative-apis-with-popular-tools.mdx

| [LlamaIndex](#llamaindex-advanced-rag-applications) | Library for building advanced AI RAG applications | Knowledge graph building, document retrieval, data indexing | Medium |
| [Continue Dev](#continue-dev-ai-coding-assistance) | IDE extension for AI-powered coding assistance | Code completion, code review | Low |
| [Zed AI](#zed-ai-coding-assistance) | IDE including AI-powered coding assistance | Code completion, code review | Low |

> | [Bolt.diy](#boltdiy-code-generation) | Software to create applications | Code generation, code edition| Low |

Reviewer: ```suggestion
| [Bolt.diy](#boltdiy-code-generation) | Software to create applications | Code generation, code edition | Low |
```

  Refer to our dedicated documentation for [installing and configuring Chatbox AI with Generative APIs](/tutorials/configure-chatboxai-with-generative-apis/)
</Message>

## Bolt.diy (code generation)

Bolt.diy is a software enabling user to create applications from prompt. 

To use Bolt.diy with Scaleway's Generative APIs, you can:
- [Install and launch Bolt.diy locally](https://github.com/stackblitz-labs/bolt.diy?tab=readme-ov-file#setup)

> - In Bolt interface, click on **Settings** in the bottom left corner.

Reviewer: ```suggestion
- In Bolt interface, click the icon next to the Bolt logo to display the sidebar.
- Click on the cogwheel icon to open **Settings** in the bottom left corner.
```

> - Then, click on **Local Providers**.

Reviewer: ```suggestion
- Click **Local Providers** to add a provider configuration.
```

- Click the cogwheel icon in the bottom left corner of the sidebar to open **Settings**.
- Click **Local Providers** to add a provider configuration.
- Toggle the switch next to **OpenAILike** to enable it and add the Scaleway API endpoint: `https://api.scaleway.ai/v1` as base URL.
- In Bolt main menu, you can now select `OpenAILike` and add your **Scaleway Secret Key** as `OpenAILike API Key`.
- You can now also select one of the supported models from Scaleway Generative APIs. We suggest you start with `llama-3.1-8b-instruct` as Bolt requires significant amount of output tokens (8000 by default), which can take a long time to generate and may not be supported by bigger models.

You can now enter a prompt and see your application being generated.

Alternatively, you can also setup your Scaleway Secret Key by renaming `.env.example` to `.env`, adding corresponding environment variables values and restarting Bolt.diy:
```bash
OPENAI_LIKE_API_BASE_URL=https://api.scaleway.ai/v1
OPENAI_LIKE_API_KEY=###SCW_SECRET_KEY###
```

To enable configuration from `.env` is taken into account, you still need to enable **OpenAILike** from Bolt interface in **Settings > Local Providers** section.

> If you first configured credentials through your browser, you also need to reset cookies from your browser for you localhost domain (as Bolt.diy stores model provider configuration information in browser).

Reviewer: ```suggestion
<Message type="note"
  If you first configured your credentials through the browser, you also need to reset your browser's cookies for your localhost domain (as Bolt.diy stores model provider configuration information in the browser).
</Message>
```

  Refer to our dedicated documentation for [installing and configuring Chatbox AI with Generative APIs](/tutorials/configure-chatboxai-with-generative-apis/)
</Message>

## Bolt.diy (code generation)

> Bolt.diy is a software enabling users to create web applications (Node.js / React) from the prompt. 

Reviewer: ```suggestion
Bolt.diy is a software enabling users to create web applications from the prompt. 
```
